File: Entities/EmailCliente.cs
namespace Entidades
{
    using System;

    public class EmailCliente
    {
        private int varId_Persona;
        private string varEmail;
        private string varDetalle;

        public int Id_Persona
        {
            get => 
                this.varId_Persona;
            set => 
                this.varId_Persona = value;
        }

        public string Email
        {
            get => 
                this.varEmail;
            set => 
                this.varEmail = value;
        }

        public string Detalle
        {
            get => 
                this.varDetalle;
            set => 
                this.varDetalle = value;
        }
    }
}

File: Entidades/EmailCliente.vb
Public Class EmailCliente
    'Variable extraída de la tabla PERSONA y que específica a cual persona pertenece el email
    Private varId_Persona As Integer
    Public Property Id_Persona() As Integer
        Get
            Return varId_Persona
        End Get
        Set(ByVal value As Integer)
            varId_Persona = value
        End Set
    End Property

    'Variable que almacena el email.
    Private varEmail As String
    Public Property Email() As String
        Get
            Return varEmail
        End Get
        Set(ByVal value As String)
            varEmail = value
        End Set
    End Property

    'Cadena que almacena alguna observación o detalle del email.
    Private varDetalle As String
    Public Property Detalle() As String
        Get
            Return varDetalle
        End Get
        Set(ByVal value As String)
            varDetalle = value
        End Set
    End Property
End Class
